### ISSUE TITLE
`%HOME%/Juris-M` is now `%HOME%/Jurism` and `\AppData\Roaming\Zotero` is `\AppData\Roaming\Jurism`

### ISSUE THREAD

**Issue by galantra** [Juris-M/zotero#127]
I installed 6.0.22m3; previous installed version was probably 6.0.22m2. I opened Juris-M and noticed that the whole database was gone, it was as if I had never ever had an installation.

I located the jurism.sqlite file and noticed that there are two folders in %HOME%:

1. my old folder, named `Juris-M`
2. a new folder named `Jurism` that was created on the day when I installed 6.0.22m3.

Then I created a symbolic link from `%HOME%/Jurism` to `%HOME%/Juris-M`. I opened Juris-M and noticed that while my database was back, my custom theme, columns and sorting were reset. 

I saw that %APPDATA% has a new folder created on the day when I installed 6.0.22m3, too. I made a symbolic link again. Now everything is as before.

# Suggested fix

I suggest that the installation of the newest versions automatically move data from the old folders to the new folders.

**Comment by fbennett** [comment]
Yes, this is alarming and needs to be addressed. I'll look at automating the move, but as an immediate fix, we can put an explanation of the change in directory name in the first-run notice that appears in the center pane of Jurism when the database is empty.
